### DISCUSSION TITLE
Bundling Wires

### DISCUSSION THREAD

**Discussion by ArsenalTheProtogen** [SebLague/Digital-Logic-Sim#189] [Ideas]
An idea that I feel like would be really useful would be the ability to bundle wires. This would be able to make chips much smaller if there was a way to convert say 8 wires into a singular large wire, so that it could use a single data line instead of having to run 8 individually. 
It could display as a pill shaped input/output as opposed to the circular ones to differentiate between the two, as well as the ability to show the values when hovering over it. This would significantly reduce space taken up as it would reduce inputs and outputs by an incredibly large amount.

**Comment by jerrecode** [comment]
Yeah having the ability to use BUS Inputs, BUS Outputs, BUS wires and resizeable DEMUX and MUX would be epic!

It is just tedious and kills all the fun an hour into the game to connect 16 Bits and 3 Control Pins for the 6th time.

**Reply by zedonme** [reply]
I use serial transmission in my chips. It takes a little more planning, but it is a great time saver

**Reply by jerrecode** [reply]
Could you share a copy of your game files i have tried many times to implement the same and i ounce even had it working but it semed to only be working if the send signal is only active for the shortest duration possible wich was pretty annoying.

Or pictures of the chips amd subchips with a short explanation?

Would be really thankful!
